modules/theme-tools/compat/twentytwentyone.php

```diff
@@ -10,6 +10,21 @@
  * Add Jetpack extra functionality to Twenty Twenty One.
  */
 function twentytwentyone_jetpack_setup() {
+
+	/**
+	 * Add theme support for Infinite Scroll.
+	 */
+	add_theme_support(
+		'infinite-scroll',
+		array(
+			'type'      => 'click',
+			'container' => 'main',
+			'wrapper'   => false,
```

Comment:
Is this necessary? I'm asking because it seems to cause the pagination not to get updated properly (if you trigger Infinite Scroll `/page/2` does not get added).

Reply:
Sorry for the delay @jeherve I originally removed that because formatting the post dividers to match at the ending was not working out for me. After some more tinkering I was able to work it out and committed the changes.

Now the wrapper is used and the dividers are consistent.